cpp/cmake/Modules/FindcuFile.cmake

```diff
@@ -93,6 +93,12 @@ find_package_handle_standard_args(cuFile
     cuFile_VERSION
 )
 
+if (cuFile_FOUND)
```

Comment:
You will want this to be:
```if(cuFile_FOUND AND NOT TARGET cuFile::cuFile_interface)```

That will allow multiple `find_package(cuFile)` to work correctly ( otherwise they will error out ).